generate_ssdlc_report_test.py

```diff
@@ -28,24 +28,24 @@ def test_report_generation():
     generate_ssdlc_report.generate_ssdlc_report(ignore_sbom_download_errors=True)
 
     # Then
-    assert os.path.exists(f"{current_directory}/ssdlc-report/MEKO-{current_version}")
-    assert os.path.exists(f"{current_directory}/ssdlc-report/MEKO-{current_version}/Containerized MongoDB Agent")
-    assert os.listdir(f"{current_directory}/ssdlc-report/MEKO-{current_version}/Containerized MongoDB Agent") != []
-    assert os.path.exists(f"{current_directory}/ssdlc-report/MEKO-{current_version}/Containerized OpsManager")
-    assert os.listdir(f"{current_directory}/ssdlc-report/MEKO-{current_version}/Containerized OpsManager") != []
-    if os.path.exists(f"{current_directory}/ssdlc-report/MEKO-{current_version}/Enterprise Kubernetes Operator"):
+    assert os.path.exists(f"{current_directory}/ssdlc-report/MCK-{current_version}")
+    assert os.path.exists(f"{current_directory}/ssdlc-report/MCK-{current_version}/Containerized MongoDB Agent")
+    assert os.listdir(f"{current_directory}/ssdlc-report/MCK-{current_version}/Containerized MongoDB Agent") != []
+    assert os.path.exists(f"{current_directory}/ssdlc-report/MCK-{current_version}/Containerized OpsManager")
+    assert os.listdir(f"{current_directory}/ssdlc-report/MCK-{current_version}/Containerized OpsManager") != []
+    if os.path.exists(f"{current_directory}/ssdlc-report/MCK-{current_version}/Enterprise Kubernetes Operator"):
         assert (
-            os.listdir(f"{current_directory}/ssdlc-report/MEKO-{current_version}/Enterprise Kubernetes Operator") != []
+            os.listdir(f"{current_directory}/ssdlc-report/MCK-{current_version}/Enterprise Kubernetes Operator") != []
         )
     assert os.path.exists(
-        f"{current_directory}/ssdlc-report/MEKO-{current_version}/SSDLC Containerized MongoDB Agent {current_version}.md"
+        f"{current_directory}/ssdlc-report/MCK-{current_version}/SSDLC Containerized MongoDB Agent {current_version}.md"
     )
     assert os.path.exists(
-        f"{current_directory}/ssdlc-report/MEKO-{current_version}/SSDLC Containerized MongoDB Enterprise Kubernetes Operator {current_version}.md"
+        f"{current_directory}/ssdlc-report/MCK-{current_version}/SSDLC Containerized MongoDB Enterprise Kubernetes Operator {current_version}.md"
```

Comment:
Should we also change these strings?
```suggestion
        f"{current_directory}/ssdlc-report/MCK-{current_version}/SSDLC Containerized MongoDB Controllers for Kubernetes {current_version}.md"
```